src/main/webapp/jsp/adminHome.jsp

```diff
@@ -6,11 +6,34 @@
 
 <c:set var="jsIncludes">
     <script type="text/javascript" src="/js/administrator.js"></script>
+    <script type="text/javascript" src="/js/adminHome.js"></script>
```

Comment:
Since these functions are used only in the AdminHome page and I don't want other pages to include them so I created a new file for them. Just like `adminAccountManagement.js`, `adminActivityLog.js` and so on. I guess it is a valid reason?